app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,34 @@
+class PagesController < ApplicationController
+  def home
+  end
+    
+  def index 
+    @journal_entry = JournalEntry.all
+  end
+
+  def new_entry_form
+    @entry = JournalEntry.new
+  end
+
+  def entries;end
+
+  def goals;end
+
+  def profile;end
+
+  def create_entry
```

Comment:
```suggestion
  def create
```

Reply:
if i later want to create tasks and goals wont i be easyer with this naming ?